src/liblzma/check/crc_common.h

```diff
@@ -0,0 +1,239 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+/// \file       crc_common.h
+/// \brief      Some functions and macros for CRC32CLMUL and CRC64CLMUL
```

Comment:
It has macros for the generic versions too so "for CRC32 and CRC64" would be better.